accounts-db/src/ancient_append_vecs.rs

```diff
@@ -755,8 +737,15 @@ impl AccountsDb {
         let mut target_slots_sorted = Vec::with_capacity(len);
 
         // `shrink_collect` all accounts in the append vecs we want to combine.
-        // This also unrefs all dead accounts in those append vecs.
-        // This needs to serially iterate largest to smallest slot so that we unref older dead slots after we have visited the newer alive slots.
+        // The following comments are no longer correct?
```

Comment:
i think these 2 comments need to be removed. ironically, we can now re-add the par_iter here if we want.

Reply:
done!